papers/bib.py

```diff
@@ -809,6 +809,10 @@ def main():
     parser = argparse.ArgumentParser(description='library management tool')
     subparsers = parser.add_subparsers(dest='cmd')
 
+    # Direct arg parsers, not in any group.
+    # =============
+    parser.add_argument('--version', action='store_true', help='Print version string, help info and exit.')
```

Comment:
Print version string and exit.

It's unusual to print help upon `--version` prompt. See for instance:
`python3 --version`
> Python 3.10.6

`git --version`
> git version 2.34.1

Reply:
To have the help printing the way to go is:
`papers --help` or `papers -h` (which is already implemented by default by argparser)

Reply:
and possibly add `papers` in front, like `python3 --version`, but it does not really matter

Reply:
Fair; set in https://github.com/boyanpenkov/papers/commit/2809f176791034adc7fb0f6eec5dfaee88a6eff3